### DISCUSSION TITLE
Getting ERROR while refactoring code to PHP8

### DISCUSSION THREAD

**Discussion by mritunjayy-tiwari** [rectorphp/rector#8174] [Q&A] [answered]
I am refactoring my codebase to upgrade to PHP 8.2.
While most of code refactoring is done by rector successfully, some files are giving following error
                                                                                                                        
```
 [ERROR] Could not process                                                                                              
         "/Users/mj.code/vagrant/sandbox/lib/Framework/Session/SessionManager.php" file,
         due to:                                                                                                        
         "System error: "Syntax error, unexpected T_STATIC91"                                                           
         Run Rector with "--debug" option and post the report here: https://github.com/rectorphp/rector/issues/new". On line: 56   
```                                                     
 
All the files which are having issue show same error (and same line number). At line 56, there is blank space.
Here is config which I am using :
<?php

declare(strict_types=1);

use Rector\CodeQuality\Rector\Class_\InlineConstructorDefaultToPropertyRector;
use Rector\Config\RectorConfig;
use Rector\Set\ValueObject\LevelSetList;

return static function (RectorConfig $rectorConfig): void {
    $rectorConfig->paths([
        __DIR__ . '/lib',
 
    ]);

    // register a single rule
    //$rectorConfig->rule(InlineConstructorDefaultToPropertyRector::class);

    // define sets of rules
       $rectorConfig->sets([
           LevelSetList::UP_TO_PHP_82
       ]);
};

**Comment by samsonasik** [comment]
we need reproduced code and rule that cause the error at https://getrector.com/demo, or sample reproducible repo

**Reply by samsonasik** [reply, marked as answer]
It should already resolved at PR:

- https://github.com/rectorphp/rector-src/pull/5001

if the issue persist, we need reproduced code and rule that cause the error at https://getrector.com/demo, or sample reproducible repo
